libraries/tools/kotlin-symbol-processing-api/src/org/jetbrains/kotlin/ksp/symbol/KSPropertyDeclaration.kt

```diff
@@ -33,6 +34,11 @@ interface KSPropertyDeclaration : KSDeclaration {
      */
     val type: KSTypeReference?
 
+    /**
+     * True if this property is mutable.
+     */
+    val isMutable: Boolean
```

Comment:
`isVar` seems more consistent with other usages. E.g., `val x: MutableMap` is mutable but not var. `isVar` sounds like the symbol can point to other objects, while `isMutable` sounds like the object can be altered.